static/bidder-info/connatix.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: "https://capi.connatix.com/rtb/ortb"
+maintainer:
+  email: "pubsolutions@connatix.com"
+gvlVendorID: 143
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - video
+userSync:
+  iframe:
+    url: "https://capi.connatix.com/us/pixel?pId=53&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&callback={{.RedirectURL}}"
+    userMacro: "[UID]"
+  redirect:
+    url: "https://capi.connatix.com/us/pixel?pId=52&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&callback={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
only iframe works:
![image](https://github.com/user-attachments/assets/c5ed1d1f-07d1-4dea-b3b6-eac6c550f608)

Reply:
thanks for the feedback! should be fixed now